generator/templates/Default/src/templates/Default.vue

```diff
@@ -1,62 +1,68 @@
 <template>
-    <div id="app">
-        <v-app id="inspire" v-if="$store.getters['Authorisation/isLoggedIn']">
-            <v-navigation-drawer
-                app
-                fixed
-                v-model="menu"
-            >
-                <img :src="require('../assets/logo.png')" @click="$router.push({name: 'home'})" class="logo">
-                <v-divider class="mt-10"/>
-                <main-menu/>
-            </v-navigation-drawer>
-            <v-app-bar app color="primary" dark mfixed>
-                <v-app-bar-nav-icon @click.stop="menu = !menu"></v-app-bar-nav-icon>
-                <v-toolbar-title>Beheer</v-toolbar-title>
-                <v-spacer/>
-                <profile-menu></profile-menu>
-            </v-app-bar>
-            <v-content>
-                <router-view/>
-            </v-content>
-            <v-footer app color="primary" inset>
-                <span class="white--text">Created by Kings Code</span>
-            </v-footer>
-        </v-app>
+    <div v-if="isLoggedIn">
```

Comment:
In welk  bestandje is nu de ` id="app"` gebleven? Ik kan het niet zo snel terugvinden